dpks/quantification.py

```diff
@@ -22,46 +22,62 @@
 
 class TopN:
     top_n: int
-    protein_nodes: List[str]
 
-    def __init__(self, top_n: int = 1):
+    def __init__(self, top_n: int = 1, level: str = "protein"):
         self.top_n = top_n
         self.num_proteins = 0
         self.num_samples = 0
-        self.protein_nodes = []
 
-    def quantify(self, quantitative_data: QuantMatrix) -> pd.DataFrame:
-        protein_quantifications = dict()
+        self.level = level
+
+    def quantify(self, quant_matrix: QuantMatrix) -> pd.DataFrame:
+        quantifications = dict()
+
+        if self.level == "protein":
+            key = "Protein"
+            entities = quant_matrix.proteins
```

Comment:
Naming inconsistent in module (see MaxLFQ). Consider changing this to ```group_ids``` and using ```group``` instead of ```entity```

Reply:
True, will change for consistency.